src/scenarios/scenarios_3.ts

```diff
@@ -1,12 +1,52 @@
 import {SubstrateService} from "../substrate-service";
 import {ScenarioInterface} from "./scenario-interface";
+import API from '@polkadot/api';
 
 export class Scenarios_3 implements ScenarioInterface {
   async run() {
     /*
        TODO: Implement Scenario 3 [Batch processing]:
          * Create 3 user wallets
-         * Sends assets to those wallets using batch transaction using link: https://github.com/Cerebellum-Network/private-standalone-network-node/blob/feature/fee-abstraction-doc/docs/batch_processing.md#2-using-polkadot-utilitybatch-method
+         * Sends assets to those wallets using batch transaction using link: https://github.com/Cerebellum-Network/private-standalone-network-node/blob/dev/docs/batch_processing.md
```

Comment:
Let's add a short scenarios description in README and remove the comment from here.

Reply:
Added to readme